modules/fbx/fbx_parser/CREDITS.lic

```diff
@@ -60,7 +60,7 @@ The GUY who performed some of the CSM mocaps.
 Contributed fixes for the documentation and the doxygen markup
```

Comment:
The `.lic` extension is weird and will likely confused some OSes. Why not keep the original name?